english/hands/door/door.md

```diff
@@ -2,4 +2,7 @@ After having watched MacGyver in your childhood, you are able to build up a door
 with the lighter.
 
 Finally, the marshmallows start to melt down and you see a beautiful beach with two moons in the sky. You have travel
-through space and time without knowing it!
\ No newline at end of file
+through space and time without knowing it!
+
+You open a door and there is an infinite, black abyss that sucks you in. You twist and contort out of control. Your clothes
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.